.github/workflows/format-checker.yml

```diff
@@ -12,14 +12,16 @@ jobs:
         submodules: true
     - name: Install dependencies from APT
       run: |
-        sudo apt install -y git clang-format-11
-        sudo update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/clang-format-11 100 \
-          --slave /usr/share/man/man1/clang-format.1.gz clang-format.1.gz /usr/share/man/man1/clang-format-11.1.gz
+        wget https://apt.llvm.org/llvm.sh
+        chmod +x llvm.sh
+        sudo ./llvm.sh 18
+        sudo apt install -y git clang-format-18
+        sudo update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/clang-format-18 100
     - name: Check Formatting
       run: |
         cd ${{ github.workspace }}
         git reset --soft $(git merge-base HEAD origin/master)
-        diff=$(git clang-format-11 --diff)
+        diff=$(git clang-format-18 --style=file --diff)
```

Comment:
I think you were right about -`-style=file`, it should be the default, so I think no need for it here unless you want to be really clear that we're formatting from a file

Reply:
Yeah, I figured we might as well be explicit about it. Saves us the trouble of going and finding the docs next time we forget if that's the default or not

Reply:
Cool, looks good then!